imports/ui/App.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import { Session } from 'meteor/session';
+import { Accounts } from "meteor/accounts-base";
+
+export default class App extends Component {
+	
+	constructor(props) {
+		super(props);
+
+		this.handleResetPassword = this.handleResetPassword.bind(this);
+		this.handlePassword = this.handlePassword.bind(this);
+		this.handleConfirmPassword = this.handleConfirmPassword.bind(this);
+
+		let resetPasswordToken = Session.get('resetPasswordToken');
+		let verifyEmailToken = Session.get('verifyEmailToken');
+
+		if (verifyEmailToken) {
+			Accounts.verifyEmail(verifyEmailToken, error => {
+				if (!error) {
+					this.setState({ emailVerified: true })
+				} else {
+					this.setState({ emailVerifyFailed: true, reason: error.reason })
+				}
+			});
+		}
+
+		this.state = { verifyEmailToken, resetPasswordToken };
+	}
+	
+	handlePassword(event) {
+		this.setState({ newPassword: event.target.value });
+	}
+
+	handleConfirmPassword(event) {
+		this.setState({ confirmPassword: event.target.value });
+	}
+
+	renderResetPasswordForm() {
+		return (
+			<form onSubmit={this.handleResetPassword}>
+				New Password:<br/>
```

Comment:
Is it possible to use CSS to make sure they're on new lines instead of using `<br/>`?